File: c++_backup/Amlyan/bala/2.cpp
#include<iostream>
using namespace std;

class a_
{
	//public:
	int a,b;
	public:
		void put(int k,int l)
		{
			a=k; b=l;
		}
		/**void add()
		{
			
		}**/
		a_()
		{
			a=1;
			b=2;
		}
		void show()
		{
			cout<<"a = "<<a<<"b = "<<b<<endl;
		}
	
};


int main()
{	
	a_ a1,b1,c1;
	//a b1;
	//a c1;
	a1.put(50,100);
	a1.show();
	b1.show();
	//c1 = a1+b1;
	//c1.a = a1.a+b1.a;
	//c1.b = a1.b+b1.b;
	c1.show();	
	return 0;
}

File: python_gui_tkinter/KALU/GARBAGE1/2.py
from tkinter import *
master = Tk()
var = []
row_no = 0
for i in range(25):
    var.append(IntVar())
for i in range(25):
	Checkbutton(master, text="male", variable=var[i]).grid(row=row_no, sticky=W)
	row_no = row_no + 1
def show():
	#row_no = 0
	i=0
	for i in range(25):
		print(var[i].get())
		#row_no = row_no + 1
Button(master, text='show selected', command=show).grid(row=row_no+1, column=2, sticky=W, pady=4)
mainloop()

